protos/semgrep_studio_v1.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package semgrep_studio_v1;
+
+option go_package = "semgrep_studio.v1";
```

Comment:
why do we need this? Can you add a comment about it?
Do we use Go somewhere at Semgrep Inc. ?

Reply:
Yes, `semgrep-app` is configured to generate golang types, and `go_package` is a requirement for that to work. It might be simpler to just exempt this file, though

Reply:
Do we use Go in semgrep-app??